bindings/matrix-sdk-ffi/src/session_verification.rs

```diff
@@ -60,80 +73,130 @@ pub struct SessionVerificationController {
 
 #[matrix_sdk_ffi_macros::export]
 impl SessionVerificationController {
-    pub async fn is_verified(&self) -> Result<bool, ClientError> {
```

Comment:
Was this method removed by mistake @stefanceriu ? We are using it in EXA.

Reply:
Oh, it was intentional, you should be using `client.encryption().verificationState()` 🤔

Reply:
OK, that's what I have done, thanks for confirming!